src/containers/shared/components/Transaction/AMM/AMMSharedSimple.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
```

Comment:
I don't like the idea of the AMM transactions all using the same Simple page. I don't think they have enough in common to justify that.

Reply:
we were sharing the assets between a lot of transactions but i need to change withdraw and deposit so I'll create the individual pages

Reply:
i.e. vote and create have the exact same simple.